ci/tests/upgrade-client.sh

```diff
@@ -1,15 +1,18 @@
 #!/bin/bash
 set -euo pipefail
 
-echo "Testing client upgradability..."
+echo "Test client upgradability of ibc-0 on basecoin-0"
 
-hermes tx upgrade-chain --reference-chain ibc-0 --host-chain basecoin-0 --host-client 07-tendermint-0 --amount 10000000 --height-offset 65
+hermes tx upgrade-chain --reference-chain ibc-0 --host-chain basecoin-0 --host-client 07-tendermint-0 --amount 10000000 --height-offset 35
 gaiad --node tcp://localhost:26657 tx gov vote 1 yes --home $HOME/data/ibc-0/data --keyring-backend test --keyring-dir $HOME/data/ibc-0 --chain-id ibc-0 --from validator --yes
-
-sleep 5s
-
+sleep 3s
 plan_height=$(gaiad --node tcp://localhost:26657 query gov proposal 1 --home $HOME/data/ibc-0/data | grep ' height:' | awk '{print $2}' | tr -d '"')
-
 echo "Waiting for upgrade plan to execute at height $plan_height..."
+hermes upgrade client --host-chain basecoin-0 --client 07-tendermint-0 --upgrade-height $plan_height
+
+echo "Test client upgradability of basecoin-0 on ibc-0"
+
+hermes tx upgrade-chain --reference-chain basecoin-0 --host-chain ibc-0 --host-client 07-tendermint-0 --amount 10000000 --height-offset 20
 
-hermes upgrade client --host-chain basecoin-0 --client 07-tendermint-0 --upgrade-height $plan_height
\ No newline at end of file
+### TODO: uncomment below when we could extract the plan height
+# hermes upgrade client --host-chain basecoin-0 --client 07-tendermint-0 --upgrade-height $plan_height
```

Comment:
This should be 

```sh
hermes upgrade client --host-chain ibc-0 --client 07-tendermint-0 --upgrade-height $plan_height
```

right?

Shouldn't we do this in this PR? Seems critical to test that our whole upgrade thing is compatible with gaiad

Reply:
Ahh, yes, Fixed to `ibc-0`

Yes, indeed should be included, though there is a blocker. Perhaps we can get it completed after that. See https://github.com/informalsystems/basecoin-rs/pull/101#discussion_r1202793112